src/components/pages/JudgeTable/JudgeTable.css

```diff
@@ -67,8 +67,9 @@
 }
 
 /* judgeTable Toolbar styles */
+/* these are all inside of CaseTable.css, as they are named the same */
 
-.menuContainer {
+/* .menuContainer {
   color: #215589;
```

Comment:
Do we need all of these commented out selectors in here still?

Reply:
Agree w/Niki, I see some commented out code that I'm not sure if it should be there or not.